backend/.env.example

```
@@ -1,4 +1,6 @@
 DATABASE_URL=postgresql+asyncpg://postgres:postgres@db:5432/invoice_db
 OPENAI_API_KEY=your_openai_api_key_here
 OPENAI_MODEL=gpt-4o-mini
+# Optional: require token to configure LLM via API
+LLM_CONFIG_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_CONFIG_TOKEN key should go before the OPENAI_API_KEY key

(UnorderedKey)